README.md

```diff
@@ -108,21 +108,38 @@ optional arguments:
 
 ### Graphic
 
-The `graphic` sub-command will create either a comprehensive diagram showing ontology modules together with classes, object properties and "Things" together with the path of imports, or (if the 'wee' option is selected) a simple diagram of ontology modules and the import hierarchy.  Graphics are exported both as ```png``` files and also as a ```dot``` file.  This ```dot``` file can be used with Graphviz or with web tools such as [Model Viewer](http://www.semantechs.co.uk/model-viewer)
+The `graphic` sub-command will create either 
+* a comprehensive diagram showing ontology modules together with classes, object properties and "Things"
```

Comment:
Suggest replace "Things" with "Individuals". Instances of user-created classes are always instances of owl:NamedIndividual.

Reply:
Done